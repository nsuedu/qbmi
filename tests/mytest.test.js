import React from 'react';
import { Button } from 'antd';
import { shallow, configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Wallet', () => {
  const wallet = mount(<Button>test</Button>);

  it('renders properly', () => {
    expect(wallet).toMatchSnapshot();
  });

  it('display the balance from props', () => {
    expect(wallet.find('.ant-btn').text()).toEqual('test');
  });
});
