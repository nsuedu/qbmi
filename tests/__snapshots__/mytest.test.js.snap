// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wallet renders properly 1`] = `
<Button
  block={false}
  ghost={false}
  htmlType="button"
  loading={false}
>
  <Wave>
    <button
      className="ant-btn"
      onClick={[Function]}
      type="button"
    >
      <LoadingIcon
        existIcon={false}
        loading={false}
        prefixCls="ant-btn"
      >
        <CSSMotion
          motionName="ant-btn-loading-icon-motion"
          onAppearActive={[Function]}
          onAppearStart={[Function]}
          onEnterActive={[Function]}
          onEnterStart={[Function]}
          onLeaveActive={[Function]}
          onLeaveStart={[Function]}
          removeOnLeave={true}
          visible={false}
        >
          <DomWrapper />
        </CSSMotion>
      </LoadingIcon>
      <span
        key=".0"
      >
        test
      </span>
    </button>
  </Wave>
</Button>
`;
